src/o365/graph/commands/schemaextension/schemaextension-list.ts

```diff
@@ -0,0 +1,155 @@
+import config from '../../../../config';
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+import Utils from '../../../../Utils';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  status?: string;
+  owner?: string;
+  pageNumber?: string;
+  pageSize?: string;
+}
+
+class GraphSchemaExtensionList extends GraphCommand {
+  public get name(): string {
+    return `${commands.SCHEMAEXTENSION_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Get a list of schemaExtension objects created in the current tenant, that can be InDevelopment, Available, or Deprecated.';
+  }
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.status = typeof args.options.status !== 'undefined';
+    telemetryProps.owner = typeof args.options.owner !== 'undefined';
+    telemetryProps.pageNumber = typeof args.options.pageNumber !== 'undefined';
+    telemetryProps.pageSize = typeof args.options.pageSize !== 'undefined';
+    return telemetryProps;
+  }
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const filter: string = this.getFilter(args.options);
```

Comment:
The `getFilter` method should only be called when command allows unknown options. In our case we have a predefined set of filters that should be used to build up the URL.